_docs/1.0.0/usage-schema.md

```diff
@@ -0,0 +1,618 @@
+| `0`    | Number                | Digit                                                                                                                                                               |                                                                                    |
+| `#`    | Number                | Digit, zero shows as absent                                                                                                                                         |                                                                                    |
+| `.`    | Number                | Decimal separator or monetary decimal separator                                                                                                                     | [Decimal separator](#deciaml_separator) NB!<sup>[*](#pattern-parsing-star)</sup>   |
+| `-`    | Number                | Minus sign                                                                                                                                                          | [Minus sign](#minus_sign) NB!<sup>[*](#pattern-parsing-star)</sup>                 |
+| `,`    | Number                | Grouping separator                                                                                                                                                  | [Grouping separator](#grouping_separator) NB!<sup>[*](#pattern-parsing-star)</sup> |
+| `E`    | Number                | Separates mantissa and exponent in scientific notation. Need not be quoted in prefix or suffix.                                                                     |                                                                                    |
+| `;`    | Subpattern boundary   | Separates positive and negative subpatterns                                                                                                                         |                                                                                    |
+| `%`    | Prefix or suffix      | Divide by 100 on parsing                                                                                                                                            |                                                                                    |
+| `‰`    | Prefix or suffix      | Divide by 1000 on parsing                                                                                                                                           |                                                                                    |
+| `'`    | Prefix or suffix      | Used to quote special characters in a prefix or suffix, for example, "'#'#" formats 123 to "#123". To create a single quote itself, use two in a row: "# o''clock". |                                                                                    |
```

Comment:
```suggestion
| `'`    | Prefix or suffix      | Used to quote special characters in a prefix or suffix, for example, the "'#'#" pattern allows the value `#123` to be read in as the number `123`. To create a single quote itself, use two in a row: "# o''clock". |                                                                                    |
```
Let's keep in mind that this is meant for standardizing to numeric types, so it reads in formatted strings and produces numbers, not the other way around.